CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 6.0.4
+
+- Version bumping related to https://github.com/alphagov/govuk_frontend_toolkit_gem/pull/15
```

Comment:
I think the content here is fine now. There seems to be a style for referring to Pull Requests though, so I'd change this to follow that, see the `([PR #15](...` like links in the rest of the changelog.